Networking/Networking/Network/WordPressOrgNetwork.swift

```diff
@@ -0,0 +1,142 @@
+import Alamofire
+import Combine
+import Foundation
+import WordPressKit
+
+/// Class to handle WP.org REST API requests.
+///
+public final class WordPressOrgNetwork: Network {
```

Comment:
Should we update https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/NETWORKING.md#network about this new `Network` implementation?

Reply:
Good thinking 💯  Updated in 14a1e2503a267fa65d4c612d5ea9863b79f1b414.